src/celeritas/em/model/WentzelModel.cc

```diff
@@ -0,0 +1,183 @@
+#include "celeritas/io/ImportParameters.hh"
+#include "celeritas/io/ImportProcess.hh"
+#include "celeritas/mat/MaterialParams.hh"
+#include "celeritas/phys/InteractionApplier.hh"
+#include "celeritas/phys/PDGNumber.hh"
+#include "celeritas/phys/ParticleParams.hh"
+#include "celeritas/phys/ParticleView.hh"
+
+namespace celeritas
+{
+//---------------------------------------------------------------------------//
+/*!
+ * Construct from model ID, particle data, and material data.
+ */
+WentzelModel::WentzelModel(ActionId id,
+                           ParticleParams const& particles,
+                           MaterialParams const& materials,
+                           ImportEmParameters const& em_params,
+                           SPConstImported data)
+    : imported_(data,
+                particles,
+                ImportProcessClass::coulomb_scat,
+                ImportModelClass::e_coulomb_scattering,
+                {pdg::electron(), pdg::positron()})
+{
+    CELER_EXPECT(id);
+
+    ScopedMem record_mem("WentzelModel.construct");
+    HostVal<WentzelData> host_data;
+
+    // This is where the data is built and transfered to the device
+    host_data.ids.action = id;
+    host_data.ids.electron = particles.find(pdg::electron());
+    host_data.ids.positron = particles.find(pdg::positron());
+
+    CELER_VALIDATE(host_data.ids,
+                   << "missing IDs (required for " << this->description()
+                   << ")");
+
+    // Electron mass
+    host_data.electron_mass = particles.get(host_data.ids.electron).mass();
+
+    // TODO: Select form factor
+    host_data.form_factor_type = NuclearFormFactorType::Exponential;
+
+    // Thomas-Fermi constant C_TF
+    const real_type ctf = fastpow(3 * constants::pi / 4, real_type{2} / 3) / 2;
+
+    // Prefactor of the screen R squared
+    host_data.screen_r_sq_elec = native_value_to<units::MevMomentumSq>(
+        em_params.screening_factor
+        * ipow<2>(constants::hbar_planck / (2 * ctf * constants::a0_bohr)));
+
+    // This is the inverse of Geant's constn
+    // need to multiply by 2 to match Geant's magic number
+    host_data.form_momentum_scale = native_value_to<units::MevMomentumSq>(
+        12 / ipow<2>(1.27 * (1e-15 * units::meter) / constants::hbar_planck)
+        / 2);
+
+    // Load Mott coefficients
+    build_data(host_data, materials);
+
+    // Construct data on device
+    data_ = CollectionMirror<WentzelData>{std::move(host_data)};
+
+    CELER_ENSURE(this->data_);
+}
+
+//---------------------------------------------------------------------------//
+/*!
+ * Particle types and energy ranges that this model applies to.
+ */
+auto WentzelModel::applicability() const -> SetApplicability
+{
+    Applicability electron_applic;
+    electron_applic.particle = this->host_ref().ids.electron;
+    // TODO: construct actual energy range
+    electron_applic.lower = zero_quantity();
+    electron_applic.upper = max_quantity();
```

Comment:
Add the lower limit for the model, something like, 
```
CELER_CONSTEXPR_FUNCTION units::MevEnergy coulomb_scattering_limit()
{
    return units::MevEnergy{1e+2};  //! 100 MeV
}
```
inside [PhysicsConstants.hh](https://github.com/celeritas-project/celeritas/blob/develop/src/celeritas/em/interactor/detail/PhysicsConstants.hh)
```suggestion
    electron_applic.lower = detail::coulomb_scattering_limit();
    electron_applic.upper = detail::high_energy_limit();
```
until we have the better way to configure these types of physics parameters.